modules/040-node-manager/images/cluster-autoscaler/werf.inc.yaml

```diff
@@ -0,0 +1,52 @@
+{{- range $key, $value := .CandiVersionMap.k8s }}
+  {{- $version := toString $key }}
+  {{- if $value.clusterAutoscalerPatch }}
+---
+image: {{ $.ModuleName }}/{{ $.ImageName }}-{{ $version | replace "." "-" }}
+from: {{ $.Images.BASE_ALPINE }}
```

Comment:
use scratch image https://github.com/deckhouse/deckhouse/wiki/How-to-build-minimal-images-from-scratch